src/Map/src/MapOptionsInterface.php

```diff
@@ -17,6 +17,13 @@
 interface MapOptionsInterface
 {
     /**
+     * @internal
+     */
+    public static function fromArray(array $array): self;
```

Comment:
We do need it I imagine ? 

Not a big fan of leaking internal methods in interfaces

Reply:
I understand your point, but that's fine if we have `@internal` no?

Reply:
I mean, interface and internal are a bit contradictory no ?

Reply:
I see what you mean, but it seems correct to me, as I want to force developers to implement `fromArray`/`toArray` if someone decide to implement a new bridge.

UX users are not supposed to see `fromArray` and `toArray` methods, they are internal to UX Map, so I believe it's fine?